lib/matplotlib/backends/backend_gtk4.py

```diff
@@ -37,6 +39,7 @@ class FigureCanvasGTK4(_FigureCanvasGTK, Gtk.DrawingArea):
 
     def __init__(self, figure=None):
         super().__init__(figure=figure)
+        print("GI version", gi.version_info)
```

Comment:
```suggestion
```

stray debugging print.

Reply:
Thanks! I guess this confirms I was testing it locally with various gi versions ;) and this did work on both lower and upper